pkgs/os-specific/linux/zfs/stable.nix

```diff
@@ -25,4 +25,12 @@ callPackage ./generic.nix args {
   version = "2.2.0";
 
   sha256 = "sha256-s1sdXSrLu6uSOmjprbUa4cFsE2Vj7JX5i75e4vRnlvg=";
+
+  extraPatches = [
+    ./brt-tunable.patch
```

Comment:
Do you think we should check this in our zfs nixos test?

```
% zpool get all | grep feature@block_cloning
zroot  feature@block_cloning          disabled                       local
```

after having the flag?

Reply:
You can definitely encode version checking inside `makeZfsTest` if you want, you just have to use `package` and look at its version.